spec/features/school_search_spec.rb

```diff
@@ -17,12 +18,13 @@
     click_on "Continue"
 
     expect(claim.eligibility.reload.claim_school).to eql schools(:penistone_grammar_school)
-    expect(page).to have_text("Are you still employed to teach at a school in England")
+    expect(page).to have_text(I18n.t("student_loans.questions.where_teaching"))
```

Comment:
Has the team discussed the preferred approach to testing localised strings? A quick browse shows our current approach to be inconsistent; sometimes the tests call the translation helper, sometimes they include the expected string as a literal. But I don't know if it's because we've never discussed it or because we have now settled on a standard but not yet updated existing code.

Reply:
Never discussed. However, these tests are not testing the copy itself, but using copy as a proxy for knowing which page we're on.